packages/suite/src/views/wallet/coinmarket/common/CoinmarketSelectedOffer/CoinmarketOfferExchange/CoinmarketOfferExchangeSendSwap.tsx

```diff
@@ -158,39 +159,44 @@ export const CoinmarketOfferExchangeSendSwap = () => {
     const { account, callInProgress, selectedQuote, exchangeInfo, confirmTrade, sendTransaction } =
         useCoinmarketFormContext<CoinmarketTradeExchangeType>();
     const [slippageSettings, setSlippageSettings] = useState(false);
-    const [slippage, setSlippage] = useState(selectedQuote?.swapSlippage || '1');
+    const selectedQuoteHelper = { ...selectedQuote };
```

Comment:
What is it for to create a copy?
As far as I can see, the object is not mutated anywhere, so why can't you use the original?

Reply:
Right, I did not notice where the mutation happens.